### PR TITLE
nerfs nitrium D:

### PR DESCRIPTION
**Pull request by Notamaniac**
# Document the changes in your pull request

Nitrium now deals 1 tox damage and makes you jitter.

Currently nitrium is easy to make compared to stimulum (which at least requires fusion temps) and has no downsides if inhaled in a low enough concentration with pluxonium (plux also being easy to make with the trit fire needed to make nitrium). This adds a downside, which shouldn't be too extreme, but enough to at least make huffing it 24/7 not an option anymore. The jitter is more to add a tell that said guy is stun immune, though I might also consider adding an examine text.

Just a reminder, nitrium is basically stimulum. (Which makes you stun immune and sleep immune and heals stamina damage faster.)

# Wiki Documentation

Nitrium deals 1.5 tox damage for every 2 points of stamina damage it heals and makes you jittery

# Changelog
:cl:  
tweak: Nitrium deals 1.5 tox damage per tick for every 2 points of stamina damage it heals and makes you jittery.
/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
> you can't really rush an actual significant ammount of nitrium at roundstart anymore, a recent PR made it require harder temps. You can only get good ammounts at 25 mins into shift at the minimum. Ided PR too. also you can just use lethals.

Found the atmos main

**Comment by Notamaniac** [comment]
> you can't really rush an actual significant ammount of nitrium at roundstart anymore, a recent PR made it require harder temps. You can only get good ammounts at 25 mins into shift at the minimum.

You do realise that a single mole is all you need right?

>also you can just use lethals.

The swarmer shoots the atmos tech with his non lethal disabler to stun him. He fails and he is told - just use lethals bro
The horror uses his tentacle to try and stun the atmos tech. He fails and he is told - just use lethals bro
The clown tries to slips the angry atmos tech chasing him with a fire axe. He fails and he is told - just use lethals bro

Just use charcoal bro :D

**Comment by iamthesilly** [comment]
Sadly due to the nature of breath code this will make it near-useless, as nitrium takes up to 8 seconds to have an effect (sometimes even more depending on server lag) and does not clear existing stuns when it does, and most fights with someone using a stun weapon like a disabler or baton will be over before it has a chance to kick in. You need to be breathing it constantly for it to be useful, and this will crit you in less than two minutes. You cannot use charcoal to counter this because it just purges the nitrium.

**Comment by Notamaniac** [comment]
> Sadly due to the nature of breath code this will make it near-useless, as nitrium takes up to 8 seconds to have an effect (sometimes even more depending on server lag) and does not clear existing stuns when it does, and most fights with someone using a stun weapon like a disabler or baton will be over before it has a chance to kick in. You need to be breathing it constantly for it to be useful, and this will crit you in less than two minutes. You cannot use charcoal to counter this because it just purges the nitrium.

Okay I made it so you only take tox damage when you have stamina damage. You should still be able to huff it, but over relying on the stun immunity will kill you sooner or later.

**Comment by iamthesilly** [comment]
> Okay I made it so you only take tox damage when you have stamina damage. You should still be able to huff it, but over relying on the stun immunity will kill you sooner or later.

I like this idea, I think you forgot to commit the change though

**Comment by DylonBR** [comment]
> Okay I made it so you only take tox damage when you have stamina damage. You should still be able to huff it, but over relying on the stun immunity will kill you sooner or later.

this is actually a great idea. would be cool if charcoal didn't purge chems that you get from internals. but good anyways

**Review by Moltijoe — APPROVED** [review]
code checks out

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1308,7 +1308,10 @@
 	return ..()
 
 /datum/reagent/nitrium_low_metabolization/on_mob_life(mob/living/carbon/M)
-	M.adjustStaminaLoss(-2 * REM, FALSE)
+	if(M.getStaminaLoss() > 0)
+		M.adjustStaminaLoss(-2 * REM, FALSE)
+		M.adjustToxLoss(1.5 *REM, FALSE)
```
Comment (ThatLing) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1313]: Why did you change it from 1 to 1.5?

```suggestion
		M.adjustToxLoss(1 * REM, FALSE)
```

**Review by ThatLing — DISMISSED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Notamaniac) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1313]: Since the tox damage only occurs when you take stamina damage, I thought to up the damage a little to compensate. Since 3 was a little too extreme, and 1 was a little too low, I went with 1.5.
